latex/pdfpc/pdfpc-doc.dtx

```diff
@@ -0,0 +1,140 @@
+%\iffalse
+%  Doc-Source file to use with LaTeX2e
+%
+%  This file is part of pdfpc.
+%  Copyright (C) 2020 Evgeny Stambulchik
+%
+%  This program is free software; you can redistribute it and/or modify
+%  it under the terms of the GNU General Public License as published by
+%  the Free Software Foundation; either version 3 of the License, or
+%  (at your option) any later version.
+%
+%  This program is distributed in the hope that it will be useful,
+%  but WITHOUT ANY WARRANTY; without even the implied warranty of
+%  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+%  GNU General Public License for more details.
+%
+%  You should have received a copy of the GNU General Public License along
+%  with this program; if not, write to the Free Software Foundation, Inc.,
+%  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+%  -------------------------------------------------------------------------------------------
+%
+%  Allow for defining some meta data and simple textual notes for use
+%  with the pdfpc presentation application <https://pdfpc.github.io>.
+%
+%  -------------------------------------------------------------------------------------------
+%
+%  Inspired by Carsten Brandt's <https://github.com/cebe/pdfpc-latex-notes>.
+%
+%  -------------------------------------------------------------------------------------------
+%
+%  Please report bugs and other problems as well as suggestions for improvements
+%  to the issue tracker at github <https://github.com/pdfpc/pdfpc/>
+%
+%  -------------------------------------------------------------------------------------------
+%\fi
+
+% !TeX spellcheck=en_US
+
+\documentclass[11pt]{ltxdoc}
+
+\usepackage{iftex}
+
+\usepackage[english]{babel}
+
+\usepackage{hyperref}
+
+\newcommand*{\sty}[1]{\textsf{#1}}
+\def\param#1{\textit{\rmfamily\mdseries\ensuremath{\langle}#1\ensuremath{\rangle}}}
+
+
+\title{The \sty{pdfpc} package \\ {\large\url{https://gitlab.com/pdfpc/pdfpc}}}
+\author{Evgeny Stambulchik}
+\date{2020/03/07 (v0.2.1)}
+
+\hypersetup{pdftitle={The pdfpc package}, pdfauthor={Evgeny Stambulchik}}
+
+\begin{document}
+\maketitle
+\thispagestyle{empty}
+
+\begin{abstract}
+This package provides a \cmd{\pdfpcnote} command for embedding notes in a way
```

Comment:
I suggest:

This package provides  a convenient way to specify notes and to define certain meta  properties  of the  presentation when used with the \href{http://pdfpc.github.io/} etc